lua_modules/file_lfs/file_lfs.lua

```diff
@@ -3,7 +3,7 @@ local FILE_READ_CHUNK = 1024
 local _file = file
 local file_exists, file_open, file_getcontents, file_rename, file_stat, file_putcontents, file_close, file_list =
   _file.exists, _file.open, _file.getcontents, _file.rename, _file.stat, _file.putcontents, _file.close, _file.list
-local node_LFS_resource = node.LFS.resource or function() end -- luacheck: ignore
+local node_LFS_resource = node.LFS.resource or function() return {} end -- luacheck: ignore
```

Comment:
This should only return `{}` if called without param. else `nil`

Reply:
Mea culpa.